ocaml/xapi/xapi_host.ml

```diff
@@ -88,7 +88,13 @@ let bugreport_upload ~__context ~host ~url ~options =
     else try Unix.getenv "http_proxy" with _ -> "" in
   let cmd = Printf.sprintf "%s %s %s" !Xapi_globs.host_bugreport_upload "(url filtered)" proxy in
   try
-    let stdout, stderr = Forkhelpers.execute_command_get_output !Xapi_globs.host_bugreport_upload [ url; proxy ] in
+    let env =
+      let path = "PATH="^(String.concat ":" Forkhelpers.default_path) in
+      match url with
+      | ""  -> [|path|] (* use default upload url in host-bugreport-upload script *)
```

Comment:
Would it work if you always pass INPUT_URL=? The empty string if the URL is empty, and let the script handle the empty case (I see that it already does).